Makefile

```
@@ -0,0 +1,21 @@
+CPYTHON_PATH               = ../cpython
+PACKAGE_ABS_PATH = $(shell pwd)/$(shell find dist/python-docs-theme-*.tar.gz)
+
+
+.PHONY: all
+all: install
```

Could we use the same target names as in the CPython, devguide and PEPs repo? Less confusion and more muscle memory :)

```suggestion
.PHONY: html
html: venv
```